src/AeroSharp/DataAccess/Internal/MapOperations.cs

```diff
@@ -54,4 +54,14 @@ IMapEntryGenerator mapEntryGenerator
     /// <returns> The map operation. </returns>
     public static Operation RemoveByKey<TKey>(string bin, TKey key, IMapEntryGenerator mapEntryGenerator) =>
         MapOperation.RemoveByKey(bin, mapEntryGenerator.GenerateKey(key), MapReturnType.KEY_VALUE);
+
+    /// <summary>
+    ///     Get the key and value rank for a map
+    /// </summary>
+    /// <param name="bin"> The bin containing the map. </param>
+    /// <param name="rank"> The rank to fetch. </param>
+    /// <param name="context"> The context within the map to apply the method to, defaults to top-level of map. </param>
```

Comment:
I was curious about this context param in the initial PR to introduce map support. I left it unexposed in order to simplify the API that AeroSharp provides, but if you think it's worth exposing should we do that across all of the methods we provide?

Reply:
Yeah, incoming in my next push!  Was just trying to figure out the best example for it 👍 

Effectively, I'm trying to get this made available: https://docs.aerospike.com/server/guide/data-types/cdt-context